capi/cbindgen.toml

```diff
@@ -4,6 +4,29 @@ language = "C"
 cpp_compat = true
 include_guard = "__blazesym_h_"
 usize_is_size_t = true
+after_includes = """\
+/* Helper macro to declare and initialize a blazesym input struct
+ *
+ * This dance with uninitialized declaration, followed by memset to zero,
+ * followed by assignment using compound literal syntax is done to preserve
+ * ability to use a nice struct field initialization syntax and **hopefully**
+ * have all the padding bytes initialized to zero. It's not guaranteed though,
+ * when copying literal, that compiler won't copy garbage in literal's padding
+ * bytes, but that's the best way I've found and it seems to work in practice.
+ *
+ * Macro declares opts struct of given type and name, zero-initializes,
+ * including any extra padding, it with memset() and then assigns initial
+ * values provided by users in struct initializer-syntax as varargs.
```

Comment:
let's leave a pointer to libbpf source code where you copied this from, both for attribution and blame distribution ;)

Reply:
Okay. I think the `memset` dance is no longer necessary with packed structs, I just forgot to remove it.